prov/gni/test/mr_notifier.c

```diff
@@ -43,9 +43,6 @@ static void mr_notifier_setup(void)
 {
 	int ret;
 
-	ret = _gnix_notifier_init(&mr_notifier);
-	cr_assert(ret == 0, "_gnix_notifier_init failed");
```

Comment:
This is still required, because we don't do library initialization in this test.